README.md

```diff
@@ -109,7 +109,8 @@ The following need to be installed.
     - If you do not have the certificate you can create a new self-signed certificate with the following command:
   `openssl req -x509 -newkey rsa:2048 -keyout key.pem -out cert.pem -days 365`
     - This will allow you to start the server but it will generate unsafe warnings in the browser. You will need a real trusted certificate for production use.
-    - Adding the `-nodes` flag will generate an unencrypted private key, allowing you to run tests without going through a password prompt
+    - Create a passphrase.txt file under config and enter in the PEM passphrase
```

Comment:
We should try to avoid the storage of our private key decryption passphrase in plaintext on the filesystem. It does not matter for development, but our production builds need to be secure! If someone were to be granted access to the contents of the passphrase file, they could decrypt all HTTPS communications.

We should explore another solution. Off the top of my head, we could get the PEM passphrase prompt to work natively, or we could just build in our own prompt.

Reply:
I agree. This was mainly a stop-gap solution for the sake of time.

Reply:
The PEM passphrase was not being prompted and I spent a lot of time trying to figure out how to make that happen, so I implemented a temporary solution.